end-to-end-test/shared/specUtils.js

```diff
@@ -327,13 +327,32 @@ function checkElementWithTemporaryClass(
 }
 
 function checkElementWithMouseDisabled(selector, pauseTime, options) {
-    return checkElementWithTemporaryClass(
+    browser.execute(function() {
+        $("<div id='blockUI'></div>")
```

Comment:
width: 100%, height:100% is implicit?

also blockUI seems not bizarre enough to never collide with anything. unlikely but not impossible enough IMO. how about using Math.random() as the id instead

Reply:
sure

Reply:
if you put something positon absolute and then do left:0, top:0, right:0, left:0, it stretches to fill portal.